README.md

```diff
@@ -1,48 +1,66 @@
-
-
-- [Schematic](#schematic)
-  - [Introduction](#introduction)
-  - [Installation Requirements and Pre-requisites](#installation-requirements-and-pre-requisites)
-  - [Package Setup Instructions](#package-setup-instructions)
-  - [Command Line Interface](#command-line-interface)
-  - [Contributing](#contributing)
-  - [Contributors](#contributors)
+[![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FSage-Bionetworks%2Fschematic%2Fbadge%3Fref%3Ddevelop&style=flat)](https://actions-badge.atrox.dev/Sage-Bionetworks/schematic/goto?ref=develop) [![Documentation Status](https://readthedocs.org/projects/sage-schematic/badge/?version=develop)](https://sage-schematic.readthedocs.io/en/develop/?badge=develop) [![PyPI version](https://badge.fury.io/py/schematicpy.svg)](https://badge.fury.io/py/schematicpy)
 
 ## Introduction
 
-SCHEMATIC is an acronym for _Schema Engine for Manifest Ingress and Curation_. The Python based infrastructure provides a _novel_ schema-based, data ingress ecosystem, that is meant to streamline the process of metadata annotation and validation for various data contributors.
+SCHEMATIC is an acronym for _Schema Engine for Manifest Ingress and Curation_. The Python based infrastructure provides a _novel_ schema-based, data ingress ecosystem, that is meant to streamline the process of dataset annotation, metadata validation and submission to an asset store for various data contributors.
 
 ## Installation Requirements and Pre-requisites
 
 * Python 3.7.1 or higher
-* [`pyenv`](https://github.com/pyenv/pyenv)
-* [`poetry`](https://github.com/python-poetry/poetry)
 
+Note: You need to be a registered and certified user on [`synapse.org`](https://www.synapse.org/), and also have the right permissions to download the Google credentials files from Synapse.
+
+## Installing
+
+Create and activate a virtual environment within which you can install the package:
+
+```
+python -m venv .venv
+source .venv/bin/activate
+```
+
+Install and update the package using [pip](https://pip.pypa.io/en/stable/quickstart/):
+
+```
+pip install -U schematicpy
+```
+
+## Package Usage Instructions
+
+The package is bundled with a Command Line client.
+
+### Schematic Initialization
+
+Use the [`init`](https://sage-schematic.readthedocs.io/en/develop/cli_reference.html#schematic-init) command to initialize the mode of authentication. This command must be run once before using any of the other CLI commands.
 
-**Important**: You need to be a registered and certified user on [`synapse.org`](https://www.synapse.org/), and also have the right permissions to download the Google credentials files from Synapse.
+```
+schematic init --config ~/path/to/config.yml
+```
 
-## Package Setup Instructions
+### Metadata Manifest Generation
```

Comment:
There are too many headers here for so little content. Maybe just have a manifest header and list the various commands in one code block using code comments to describe what each of them do.

Reply:
That said, I see that you have links, which can't be used in code block. In that case, use another format that doesn't rely on headings.